docs/data-sources/federated_docker_repository.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
I don't think this resource is necessary since it's a deprecated local repo resource.